From 15bf4096ea496a28210e2f4c1f34ab28b413566a Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Thu, 24 Aug 2017 08:52:18 -0400
Subject: [PATCH 3/4] Ignore amavisd-new "starting child processes"
 notifications.

Newer versions of amavisd-new log the following constantly:

  sd_notify (no socket): STATUS=Starting child process(es), ready for work.

It's of no use to us, so add it to the global "ignore" list.
---
 amavis-logwatch | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/amavis-logwatch b/amavis-logwatch
index 1aab787..06b60c8 100644
--- a/amavis-logwatch
+++ b/amavis-logwatch
@@ -2044,6 +2044,8 @@ sub create_ignore_list() {
         # unanchored
    push @ignore_list_final, qr/\bRUSAGE\b/;
    push @ignore_list_final, qr/: Sending .* to UNIX socket/;
+
+   push @ignore_list_final, qr/sd_notify \(no socket\): STATUS=Starting child process\(es\), ready for work./
 }
 
 # Notes:
-- 
2.13.0

